### PR TITLE
Add server run readme instructions

### PR DESCRIPTION
**Pull request by karesti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,23 @@
...
+This project needs a standalone infinispan server running locally.
+The Infinispan server exposes the [REST API](https://infinispan.org/docs/dev/titles/rest/rest.html) 
+that is used in this console.
+
+In production, this console is built as a dependency using maven. This dependency is added to the infinispan
+server bundle, so the console is served from the server in production.
+
+### Use Docker
+```bash
+ docker run -it -p 11222:11222 -e USER="user" -e PASS="pass" infinispan/server:latest
```
Comment (ryanemerson) [inline README.md:17]: FYI `latest` will get the latest stable release not dev, so if there is a feature required by the console on master it won't be available. To get the latest release of a given stream you can use `:11.0` or `10.1` etc.

Comment (karesti) [reply README.md:17]: ok

File: README.md [hunk 2]
```diff
@@ -2,7 +2,26 @@
...
+
+In production, this console is built as a dependency using maven. This dependency is added to the infinispan
+server bundle, so the console is served from the server in production.
+
+### Use Docker
+
+To run the latest development release version, run 11.0
+
+```bash
+ docker run -it -p 11222:11222 -e USER="user" -e PASS="pass" infinispan/server:11.0
```
Comment (pruivo) [inline README.md:20]: Suggestion: add `--rm` to remove the container when it exits.
```suggestion
 docker run -it --rm -p 11222:11222 -e USER="user" -e PASS="pass" infinispan/server:11.0
```

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**